qiskit/backends/ibmq/ibmqbackend.py

```diff
@@ -10,13 +10,17 @@
 This module is used for connecting to the Quantum Experience.
 """
 import logging
+import os
+import json
+import jsonschema
 
 from IBMQuantumExperience import ApiError
 from qiskit import QISKitError
 from qiskit._util import _camel_case_to_snake_case, AvailableToOperationalDict
 from qiskit.backends import BaseBackend
 from qiskit.backends.ibmq.ibmqjob import IBMQJob
 from qiskit.backends import JobStatus
+from qiskit import __path__ as qiskit_path
```

Comment:
This is probably not necessary since `qiskit.__path__[0]` or `os.path.dirname(qiskit.__file__)` is common and you only use it once.

Reply:
Shall I `import qiskit` then, so I can access `qiskit.__path__`?

Reply:
I think that sounds fine.